runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttRouteConfig.java

```diff
@@ -56,21 +62,30 @@ boolean authorized(
         return authorized.test(authorization);
     }
 
+    String identity(
+        String guard,
+        long authorization)
+    {
+        return identities.getOrDefault(guard, a -> null).apply(authorization);
+    }
+
```

Comment:
Lookup by guard name can be moved to parse time as we know the guard name when parsing the value expression.

So this needs to return the `LongFunction` instead.

```suggestion
    String identity(
        String guard)
    {
        return identities.get(guard);
    }
```

Alternatively, we could move `identities` to public scope and just use `route.identities::get` instead.

Reply:
Done, thanks!